### DISCUSSION TITLE
Add `<core><decoration>border` to disable SSD

### DISCUSSION THREAD

**Discussion by ghost** [labwc/labwc#875] [Ideas]
I would like to use window borders instead of the title bar. CSD don't fit the bill, and frankly I don't think they should exist. An in-between option that uses SSD but disables the title bar would be appreciated...

**Comment by ghost** [comment]
If the issue is as simple as I think, I would be happy to open a pull request

**Comment by johanmalm** [comment]
Is this the same as
https://github.com/labwc/labwc/issues/751

**Comment by Consolatis** [comment]
Added a comment about a potential implementation at

- #813

Suggest moving the discussion about it to that issue.
